packages/apps/human-app/frontend/src/router/components/drawer-menu-items/drawer-menu-items-worker.tsx

```diff
@@ -7,6 +7,7 @@ import {
 import { DarkModeSwitch } from '@/shared/components/ui/dark-mode-switch';
 import type { UserData } from '@/modules/auth/context/auth-context';
 import { routerPaths } from '@/router/router-paths';
+import { KycStatus } from '@/modules/worker/profile/types/profile-types';
```

Comment:
nit: here and in other places, it can be imported from `@/modules/worker/profile`